alienvault/resource_sensor.go

```diff
@@ -73,7 +73,7 @@ func resourceSensorCreate(d *schema.ResourceData, m interface{}) error {
 		return err
 	}
 
-	d.SetId(sensor.UUID)
+	d.SetId(sensor.V1ID)
```

Comment:
They changed this field name between versions, so you can just call `ID()` and it'll grab the right one:
```suggestion
	d.SetId(sensor.ID())
```